How-To-Release-Riak.md

```diff
@@ -0,0 +1,56 @@
+This doc contains the internal Basho policy for building, testing, and releasing Riak.
+
+# Scheduling
+All public releases must conform to the following schedule:
+* An expected delivery date on a Wednesday. Each delivery date must have at least one fallback date that is no earlier than one week after the expected delivery date.
```

Comment:
Let me start with, I love fallback dates.  Questions: Is a week _enough_ to smooth out all cases?  If we miss the first date, do we establish a new fallback once our fallback becomes the target?